ra/fsp/inc/instances/r_sdhi.h

```diff
@@ -68,7 +68,7 @@ typedef union
         uint32_t reserved_5         : 3;
         uint32_t card_dat3_removed  : 1; // < card removal detected by dat3 pin
         uint32_t card_dat3_inserted : 1; // < card inserion detected by dat3 pin
-        uint32_t reservered_10      : 6;
+        uint32_t reserved_10      : 6;
```

Comment:
Code change, someone from Renesas should confirm this.
- Likely fine, reserved items should not be used?